code/espurna/ota.ino

```diff
@@ -30,8 +30,22 @@ void _otaLoop() {
 
 #if TERMINAL_SUPPORT || OTA_MQTT_SUPPORT
 
-#include <ESPAsyncTCP.h>
-AsyncClient * _ota_client;
+#ifndef ARDUINO_ESP8266_RELEASE_2_5_0
+#define USING_AXTLS // do not use BearSSL
+#endif
+
+WiFiClient _ota_client;
+
+#if ASYNC_TCP_SSL_ENABLED
```

Comment:
OTA_SECURE_ENABLED?

Reply:
Maybe it's even better to use a generic ``SUPPORT_SSL`` flag (or similar)? We can use that in MQTT and other modules as well.

Reply:
Per pr - we no longer use async client, so why depend on it?

> We can use that in MQTT and other modules as well.

Yep, no reason not to enable all secure things at the same time (since they use the same library and we bundle it anyways).

Reply:
I have added SSL_ENABLED now - if it looks good I'll make the changes across the whole project in a separate PR